icu4c/source/tools/ctestfw/ctest.c

```diff
@@ -715,6 +714,7 @@ static UBool vlog_knownIssue(const char *ticket, const char *pattern, va_list ap
     } else {
       log_verbose("(Known issue %s) %s\n", ticket, buf);
     }
+    va_end(ap);
```

Comment:
I can't see any corresponding `va_start()` in this function. Is that really correct?

Reply:
It's not. I missed that one when cleaning up the commit. I'll remove it.